### PR TITLE
[Example] Fix Face Recognition UI issues

### LINKED ISSUE #1055: [Example] UI update and fix for Face Recognition
Bug List:

- [x] The "FaceReidentification" and "SSD MobileNet v1 Face" should be selected when visit http://127.0.0.1/examples/face_recognition/?prefer=none&b=WebGL&m=ssd_mobilenetv1_face_tflite+facereidentification_recognition_openvino&s=image&d=0
- [x] Section padding issue
- [x] Logos trademark issue
- [x] Wrong prompt when visit http://127.0.0.1/examples/face_recognition/?prefer=none&b=WASM&m=ssd_mobilenetv1_face_tflite&s=image&d=0 , it should display 'Not enough selected models'
- [x] Title shouldn't display "Face Recongnition / WebGL / None" when visit http://127.0.0.1/examples/face_recognition/?prefer=none&b=WebGL&m=ssd_mobilenetv1_face_tflite+facereidentification_recognition_openvino&s=image&d=0
- [x] Select image steps
- [ ] %20 shouldn't replace "+" in URL when switch to dual backends mode (won't fix, not easy to repro)
- [x] Messy Camera tab
- [x] Messy Camera mode in full screen

Other fixes needed:  - will fix it when WebNN support relevant ops.
- [ ] Links in home page and navigation bars
- [ ] Model page update

### PR DESCRIPTION
**Pull request by ibelem**
Fix #1055 

Bug List:

- [x] The "FaceReidentification" and "SSD MobileNet v1 Face" should be selected when visit http://127.0.0.1/examples/face_recognition/?prefer=none&b=WebGL&m=ssd_mobilenetv1_face_tflite+facereidentification_recognition_openvino&s=image&d=0
- [x] Section padding issue
- [x] Logos trademark issue
- [x] Wrong prompt when visit http://127.0.0.1/examples/face_recognition/?prefer=none&b=WASM&m=ssd_mobilenetv1_face_tflite&s=image&d=0 , it should display 'Not enough selected models'
- [x] Title shouldn't display "Face Recongnition / WebGL / None" when visit http://127.0.0.1/examples/face_recognition/?prefer=none&b=WebGL&m=ssd_mobilenetv1_face_tflite+facereidentification_recognition_openvino&s=image&d=0
- [x] Select image steps
- [ ] %20 shouldn't replace "+" in URL when switch to dual backends mode (won't fix, not easy to repro)
- [x] Messy Camera tab
- [x] Messy Camera mode in full screen

Other fixes needed - will fix it when WebNN backends support relevant ops.
- [ ] Links in home page and navigation bars
- [ ] Model page update

### CODE REVIEW

**Comment by ibelem** [comment]
@BruceDai PTAL.

The Circle CI still fails.

**Comment by BruceDai** [comment]
@ibelem I checked circleci failure build information from [Details](https://circleci.com/gh/intel/webml-polyfill/281?utm_campaign=vcs-integration-link&utm_medium=referral&utm_source=github-build-link). The environment for CI still used the old docker image (circleci/node:12.9.1-browsers),  not the new docker image (circleci/node:13.1.0-browsers) which I have upgraded by PR #1071. Are you able to rerun the build since I don't have the write permission to rerun.

File: examples/static/js/ui.common.js [hunk 1]
```diff
@@ -105,16 +105,26 @@ const formatToLogo = {
...
 
 const trademarks = (allFormats) => {
-  let trademarknote;
-  for (const format of allFormats) {
+  let trademarknote = '';
+
+  for (let format of allFormats) {
+    let trademark = '';
+
     if (format.toLowerCase() === 'tflite') {
```
Comment (BruceDai) [inline examples/static/js/ui.common.js]: @ibelem How about use switch statements for these three 'if (format.toLowerCase() === **format**)' blocks?

Comment (ibelem) [reply examples/static/js/ui.common.js]: Thanks @BruceDai , updated, PTAL :)

Comment (BruceDai) [reply examples/static/js/ui.common.js]: LGTM, thanks

**Pull request merged**